### DISCUSSION TITLE
Truncated incorrect DOUBLE value for string

### DISCUSSION THREAD

**Discussion by givihuda** [dbeaver/dbeaver#17159] [Q&A]
I can't tell what this is referring to:

![image](https://user-images.githubusercontent.com/74785256/180492600-a02133c3-4541-4f57-8d71-b94532c0810f.png)

The SQL query I'm testing is not my own, it's big and complicated, lots of columns, and there's several fields that are fetched and displayed that contain "MULTIPLE"... I'm not seeing anything in the query trying to handle those values as numbers. So far every column showing anything like 'MULTIPLE' is varchar.

Note that I can't just post the query because it's proprietary company information. Please assume this is the default for most people. Also I'm not looking for help fixing the query, just for how to trace what exactly the error is referring to, as DBeaver doesn't specify.

**Comment by mkgrgis** [comment]
This problem isn't about DBeaver. It's about a database executing the query.

**Reply by givihuda** [reply]
The problem is DBeaver gives me no information on where the problem is happening... I'm not asking anyone to fix my SQL query or database structure or data; I'm trying to get more information and DBeaver is just saying "Here's a kind of problem" but not giving me any way to trace it.

Where did it identify the problem? What do I click to get a trace to the portion of the SQL query or even which table column it's referring to?

Can someone answer that understands what I'm asking please?

**Reply by mkgrgis** [reply]
> Where did it identify the problem?

It's not about "Output" especially for Oracle. No diagnostics of Your problem if no usual context:

> #### System information: 
> - Operating system (distribution) and version
> - DBeaver version
> - Additional extensions
> 
> #### Connection specification:
> - Database name and version
> - Driver name
> - Do you use tunnels or proxies (SSH, SOCKS, etc)? 
> 
> #### Describe the problem you're observing:
> 
> #### Steps to reproduce, if exist:

**Reply by givihuda** [reply]
It's not Oracle... it's MySQL 5.7.

I'm not sure that telling you my specs is necessary to you understanding what I'm asking... are you not aware of how the tab error in the screenshot works? Who coded it to work that way? Where would that error have come from? Does anyone have any idea how that Output tab works with that error? Somebody at DBeaver must be aware of the source code for that and can tell at least if that error message or any part of it is coming from DBeaver or MySQL. 

Are you saying it's coming straight from MySQL? OK, so from which part of a SELECT execution could it have come from? A JOIN? A pseudo-value? Can DBeaver tell? Or is it just flat output from MySQL that DBeaver can't trace? I'm trying to figure out which column/alias it's referring to. At least you should be able to to answer some of these questions in your head at minimum here.

- macOS Monterey 12.5
- DBeaver 22.1.2.202207091901
- I haven't installed any extensions. I just use it with MySQL 5.6 and 5.7 databases.

- What good is my database name and version? Do you mean MySQL 5.7? That's the version that's connected to anyway (the database server is hosted by my MAMP Pro through port 8889, though I don't know how that's relevant at the moment), though some of the database data imports are coming from MySQL 5.6 databases.
- I'm not 100% sure of all of the driver info you need, but here's a screenshot:

![image](https://user-images.githubusercontent.com/74785256/181386508-2c9e7b83-36d5-4765-828d-dc9ec93cd8a0.png)

- I don't use proxies for this database connection which is just connecting locally directly localhost:8889, then I right-click the database name (like my-database-name-thing) then select SQL Editor > New SQL Script, then I pasted the test query from the code and got that list of untraceable errors.

**I already described the problem.**

**I already described how to reproduce.**

Please re-read the thread from the beginning carefully to fully understand what's being said and asked.

**Reply by mkgrgis** [reply]
For MySQL You must test input data for corresponding datatype. Look like in INSERT for `double precision` attribute there is string literal, not d.d[e-d].

**Reply by givihuda** [reply]
I'm not sure I understand your reply; the data's already inserted. I'm getting the weird tab error from a SELECT (with multiple tables JOINed).

I think you're on the right track with "`double precision` attribute there is string literal" but I'm not sure how to trace it; I would still like to know if the specific error message is coming from MySQL output or is a DBeaver error message (coded by the DBeaver coders).

Anyway, I think I'll have to run some query tests or examinations or something, I'm not sure what yet.

I was just hoping DBeaver had some feature I could click to trace the error, or the devs here could shed more light on the Output tab in relation to this error.
